plugins/woocommerce-blocks/assets/js/atomic/blocks/product-elements/sku/block.tsx

```diff
@@ -44,15 +45,17 @@ const Preview = ( {
 		style={ style }
 	>
 		<RichText
+			className="wc-block-components-product-sku__prefix"
 			tagName="span"
-			placeholder="Prefix"
+			placeholder={ __( 'Prefix', 'woocommerce' ) }
 			value={ prefix }
 			onChange={ ( value ) => setAttributes( { prefix: value } ) }
 		/>
-		<strong>{ sku }</strong>
+		<span> { sku }</span>
 		<RichText
+			className="wc-block-components-product-sku__suffix"
 			tagName="span"
-			placeholder="Suffix"
+			placeholder={ ' ' + __( 'Suffix', 'woocommerce' ) }
```

Comment:
I added the space here to follow the same pattern that [GB is using](https://github.com/WordPress/gutenberg/blob/trunk/packages/block-library/src/post-terms/edit.js#L149).